cfgmgr/buffermgr.h

```diff
@@ -0,0 +1,54 @@
+#ifndef __BUFFMGR__
+#define __BUFFMGR__
+
+#include "dbconnector.h"
+#include "producerstatetable.h"
+#include "orch.h"
+
+#include <map>
+#include <string>
+
+namespace swss {
+
+#define INGRESS_LOSSLESS_PG_POOL_NAME "ingress_lossless_pool"
```

Comment:
Is assuming ingress lossless pg pool name the right thing to do?

Can this code support both shared ingress pg pool and separate pg pools for lossless and lossy traffic?

Reply:
currently we support buffers update for profiles which use ingress lossless pools only

Reply:
Does it prevent shared pg pool?

Reply:
I just had a discussion with Guohan, this change will affect Broadcom platform in negative way.

On some Broadcom platforms, due to the MMU memory size limitation, we have to allocate a shared buffer pool for all pgs (lossy and lossless), and create 2 buffer profiles on top of the shared pg pool.

I think by hard coding the pg pool name, here is a potential risk for Broadcom platforms that requires shared buffer pool.

There is a buffers.json file checked in for Broadcom TH chip: https://github.com/Azure/sonic-swss/blob/master/swssconfig/sample/th.64ports.buffers.json

Please make sure your code works on this scenario as well.

Thanks,
Ying

Reply:
ok, pool_name is yet another candidate to get to the new table with QOS/BUFFER configuration (besides pg/queue index to generate profile for)
But existing implementation will still work with this platform.
Since the configuration is static no new profiles will be generated on speed change.

Reply:
I've updated config for Force10-S6100 (previously by mistake used `td2.32ports.buffers.json` for it)

Reply:
I see. thanks for the update.